install_config/configuring_authentication.adoc

```diff
@@ -1004,6 +1004,10 @@ oauthConfig:
 ----
 <1> This provider name is prefixed to the GitLab numeric user ID to form an
 identity name. It is also used to build the callback URL.
+<2> When *true*, unauthenticated token requests from non-web clients (like
+the CLI) are sent a `WWW-Authenticate` challenge header for this provider.
+This uses the http://doc.gitlab.com/ce/api/oauth2.html#resource-owner-password-credentials[Resource Owner Password Credentials]
+grant flow to obtain an access token from GitLab. 
 <2> *GitLabIdentityProvider* cannot be used to send `WWW-Authenticate`
```

Comment:
Ok, I added a fix: https://github.com/openshift/openshift-docs/pull/2716/commits/ed5304b8d7d581c5925aef954f7ae558e6ad2b1d

Thank you!